Screenbox/Pages/FolderListViewPage.xaml

```diff
@@ -78,8 +78,6 @@
 
     <Grid>
         <muxc:ProgressBar
-            x:Name="LoadingProgressBar"
-            Margin="{StaticResource PageContentMargin}"
```

Comment:
@huynhsontung Can you check this spacing with the ListView? I pretty sure the ListView (like the GridView on the FolderViewPage) needs a Margin property, unfortunately I can't test it.

Reply:
The spacing looks good

![image](https://github.com/huynhsontung/Screenbox/assets/31434093/d65d0ecc-6df5-4ffc-8ce3-50437857f51a)